components/ResultsPage/index.tsx

```diff
@@ -41,8 +50,8 @@ const getEligibilityText = (
   key: string
 ): string => {
   return getEligibility(resultsArray, key)
-    ? `${apiTsln.benefit[key]}: ${apiTsln.result.eligible}`
-    : `${apiTsln.benefit[key]}: ${apiTsln.result.ineligible}`
+    ? `${apiTsln.benefit[key]}`
+    : `${apiTsln.benefit[key]}`
```

Comment:
? same result either way. you prob meant something different

Reply:
another good catch! That whole method is actually unneccesary now, so it was deleted